src/Symfony/Bundle/WebProfilerBundle/Resources/config/commands.xml

```diff
@@ -13,11 +13,13 @@
         <service id="web_profiler.command.import" class="%web_profiler.command.import.class%">
             <argument type="service" id="profiler" on-invalid="null" />
             <tag name="console.command" />
+            <deprecated>The "%service_id" service is deprecated since Symfony 2.8 and will be removed in 3.0.</deprecated>
```

Comment:
this is a bad idea. the console will always load the service when registering commands. The deprecation is shown when calling the command